### PR TITLE
adds toggle button to alt scores

### PR DESCRIPTION
**Pull request by esotrope**
alt-scores

### CODE REVIEW

File: webapp/src/main/webapp/src/app/assessments/model/score-statistics.ts [hunk 1]
```diff
@@ -1,5 +1,10 @@
 import { SubjectDefinition } from '../../subject/subject';
 
+/**
+ * Declares all the different score types
+ */
+export type ScoreType = 'Overall' | 'Claim' | 'Alternate';
```
Comment (malaffoon) [inline webapp/src/main/webapp/src/app/assessments/model/score-statistics.ts]: Can we put `Alternate` between the other two?

File: webapp/src/main/webapp/src/assets/i18n/en.json [hunk 2]
```diff
@@ -403,6 +403,7 @@
     },
     "buttons": {
       "all": "All",
+      "display-alternate": "Alt Scores",
```
Comment (malaffoon) [inline webapp/src/main/webapp/src/assets/i18n/en.json]: This label (and `"Claim Scores"`) should come from, for example, `subject.ELPAC.asmt-type.sum.alt-score.name` and `subject.ELPAC.asmt-type.sum-claim-score.name` (substituting the subject code and asmt-type code of course).

File: webapp/src/main/webapp/src/app/assessments/results/average-scale-score.component.html [hunk 3]
```diff
@@ -1,320 +1,339 @@
...
+    <button
+      *ngIf="subjectDefinition.alternateScores"
+      (click)="onScoreTypeChange('Alternate')"
+      class="btn btn-default z-index-over-data-table"
+      angulartics2On="click"
+      angularticsAction="ViewClaimScores"
+      angularticsCategory="AssessmentResults"
+      [ngClass]="{ active: scoreType === 'Alternate' }"
+    >
+      {{ 'common.buttons.display-alternate' | translate }}
```
Comment (malaffoon) [inline webapp/src/main/webapp/src/app/assessments/results/average-scale-score.component.html]: see note below about the key for this

File: webapp/src/main/webapp/src/app/assessments/results/average-scale-score.component.html [hunk 4]
```diff
@@ -1,320 +1,339 @@
...
+      (click)="onScoreTypeChange('Claim')"
       class="btn btn-default z-index-over-data-table"
       angulartics2On="click"
       angularticsAction="ViewClaimScores"
       angularticsCategory="AssessmentResults"
-      [ngClass]="{ active: isClaimScoreSelected }"
+      [ngClass]="{ active: scoreType === 'Claim' }"
     >
-      <i class="fa fa-bold fa-table"></i>
       {{ 'common.buttons.display-claim' | translate }}
```
Comment (malaffoon) [inline webapp/src/main/webapp/src/app/assessments/results/average-scale-score.component.html]: see note below about the key for this

File: webapp/src/main/webapp/src/app/assessments/model/score-statistics.ts (hunk 1 above)
Comment (malaffoon) [reply webapp/src/main/webapp/src/app/assessments/model/score-statistics.ts]: Okay, i guess the order here doesn't matter that much. But the buttons should probably have alt-scores in the middle.

Comment (esotrope) [reply webapp/src/main/webapp/src/app/assessments/model/score-statistics.ts]: yeah i can do that

File: webapp/src/main/webapp/src/assets/i18n/en.json (hunk 2 above)
Comment (esotrope) [reply webapp/src/main/webapp/src/assets/i18n/en.json]: interesting ok. sounds like a feature enhancement ; j

**Pull request merged**